src/client/dom-processor/client-dom-adapter.js

```diff
@@ -19,12 +19,7 @@ export default class ClientDomAdapter extends BaseDomAdapter {
     }
 
     hasAttr (el, attr) {
-        for (var i = 0; i < el.attributes.length; i++) {
-            if (el.attributes[i].name === attr)
-                return true;
-        }
-
-        return false;
+        return el.hasAttribute(attr);
```

Comment:
It is wrong. 
see http://jsperf.com/has-attribute-versus-iterate (change to Chart Type to table view)